syz-cluster/pkg/service/discussion.go

```diff
@@ -0,0 +1,75 @@
+// Copyright 2025 syzkaller project authors. All rights reserved.
+// Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+package service
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/google/syzkaller/syz-cluster/pkg/api"
+	"github.com/google/syzkaller/syz-cluster/pkg/app"
+	"github.com/google/syzkaller/syz-cluster/pkg/db"
+)
+
+type DiscussionService struct {
+	reportRepo      *db.ReportRepository
+	reportReplyRepo *db.ReportReplyRepository
+}
```

Comment:
Some explanation is needed here. What is its role?

Reply:
Added a comment. PTAL.